src/Network/Minio.hs

```diff
@@ -257,9 +261,22 @@ removeBucket bucket = do
 bucketExists :: Bucket -> Minio Bool
 bucketExists = headBucket
 
-
 -- | Removes an ongoing multipart upload of an object.
 removeIncompleteUpload :: Bucket -> Object -> Minio ()
 removeIncompleteUpload bucket object = do
   uploads <- listIncompleteUploads bucket (Just object) False C.$$ CC.sinkList
   mapM_ (abortMultipartUpload bucket object) (uiUploadId <$> uploads)
+
+
+-- | Fetch the policy set on a bucket.
+getBucketPolicy :: Bucket -> Minio Text
+getBucketPolicy bucket = getBucketPolicy' bucket
```

Comment:
We don't need to add a `getBucketPolicy'` prime variant. Just add `getBucketPolicy` to the export list in this file and rename the function in S3API.hs. This definition can be removed. We are just re-exporting from here.

Reply:
this is how I had before seen the previous comment from @krisis to have a lower level version as well.

Reply:
now that we are not going to add any type-safety to policies, we can just re-export these functions defined in `Network.Minio.S3API` module by the same name from `Network.Minio`. sorry about the back and forth.